### PR TITLE
Added backend build for travis

### PR DESCRIPTION
**Pull request by taerim25**
# Description

I've added travis CI for backend! Frontend build will be added once frontend setup is merged to master.
I am currently manually running all the unit test by python commands, since one of the tests automatically fail (presumably due to having lack of production directory) . If there are better ways to do this, please suggest!

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

By checking the build manually through travis website.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I will rebase before merging

### CODE REVIEW

**Comment by taerim25** [comment]
[TRAVIS: Build Backend](https://trello.com/c/y9fqT96N/24-travis-build-backend)

**Comment by Ty-Won** [comment]
@taerim25 which test is failing?

**Comment by taerim25** [comment]
@Ty-Won  python3 -m unittest discover tests/integrations is the test call where it fails.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+before_script:
+ - python3 ./scripts/setup_db.py
+script:
+ - python3 -m unittest discover tests/api
+ - python3 -m unittest discover tests/unit
+services:
+ - postgresql
+
+
+
```
Comment (yuedai0531) [inline .travis.yml]: take out these lines

**Comment by taerim25** [comment]
There seems to be merge conflict in the master, can somebody resolve it :(

**Comment by Ty-Won** [comment]
@taerim25 its being resolved! Someone is working on the fix and I'll approve of the fix asap so we can merge this pr

**Comment by Ty-Won** [comment]
@taerim25 the merge conflict was fixed!

**Pull request merged**